docs/src/toml.md

```diff
@@ -15,7 +15,7 @@ A parameter is determined by its unique name. It has possible attributes
 6. `transformation`
 
 !!! warn
-    Currently we only support `float` and `array{float}` types. (option-type flags and string switches are not considered CLIMAParameters.)
+    Currently we only support `float`, `array{float}`, `integer`, `string`, and `bool` types. (option-type flags are not considered CLIMAParameters.)
```

Comment:
I believe we support arrays of all these types. Maybe add:
"Currently we support `Type` and `array{Type}` for `Type` can be `float`,`integer`, `string` and `bool` "